### DISCUSSION TITLE
Streaming in 4K HDR 60FPS Sunshine crashing

### DISCUSSION THREAD

**Discussion by Albron96** [LizardByte/community#444] [Sunshine]
I'm running the latest sunshine server version with an RTX3080 10GB. Whenever I stream in HDR, and start any game, the stream crashes. Goes to a black screen where audio and game are still playing before completely crashing after a few seconds and having to go restart sunshine on the host PC. I tried lowering resolution and settings with no luck. SDR works great with no issues. I've seen the issue regarding vram but it's supposed to be fixed on the latest release so I'm not sure what's going on.

**Comment by iancs** [comment]
I spent a week trying to fix this by:
* installing different drivers
* turning off HAGS
* changing the priority setting in Sunshine
* fiddling with different settings in Moonlight

What fixed it for me was buying a headless HDMI dongle for $3.  I made one small mistake with that, as I bought one that didn't support HDR. I ended up using this [driver](https://github.com/itsmikethetech/Virtual-Display-Driver/releases/) to get around that (as an entirely new display). The problem with this is that the Nvidia overlay doesn't recognize it as an Nvidia-driven display so you can't use certain features with it (like autohdr).  If it bugs me enough I'm going to buy a 4k60fpsHDR HDMI plug ($6).

Just for completeness I'm using Monitor Profile Switcher to handle automatically enabling/disabling these displays.

I really hope this helps you, because I probably spent a dozen hours messing about trying to fix this.

[a comment by an automated account is not shown]
